examples/volumes/glusterfs/README.md

```diff
@@ -84,17 +87,13 @@ You can verify that the pod is running:
 $ kubectl get pods
 NAME             READY     STATUS    RESTARTS   AGE
 glusterfs        1/1       Running   0          3m
-
-$ kubectl get pods glusterfs -t '{{.status.hostIP}}{{"\n"}}'
-10.240.169.172
 ```
 
-You may ssh to the host (the hostIP) and run 'mount' to see if the Glusterfs volume is mounted,
+You may execute the command `mount` inside the container to see if the GlusterFS volume is mounted correctly:
```

Comment:
@humblec Of course this would be an option. But the reason I removed the old example in first place was that the given command `kubectl get pods glusterfs -t '{{.status.hostIP}}{{"\n"}}'` does not work (`Error: unknown shorthand flag: 't' in -t`). 

The new example works, covers the complete use-case, and is easier (only a single `exec` command, no ssh'ing into the node).